libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4267,7 +4267,7 @@ void GCS_MAVLINK::handle_heartbeat(const mavlink_message_t &msg)
 {
     // if the heartbeat is from our GCS then we don't failsafe for
     // now...
-    if (msg.sysid == gcs().sysid_gcs()) {
+    if (gcs().sysid_is_gcs(msg.sysid)) {
```

Comment:
I'm wondering about `sysid_is_mygcs` - it's old nomenclature but is a bit clearer on intent.

Reply:
for an internal API we should use the nomenclature we use elsewhere, so I think "is_gcs" is right